tools/data/add_transcripts_from_gff3_to_annotations.pl

```diff
@@ -142,6 +144,7 @@ sub print_usage {
     [--success_log|-l <success log file>]
     [--error_log|-L <error log file>]
     [--skip|-s <skip id file>]
+    [--test|-x]
```

Comment:
Perfect . . then it is correct as it is.  I just thought it was verbosely dumping the output.